docs/videos/presentations.md

```diff
@@ -1,9 +1,12 @@
 # Aragon Presentation Videos
 
+<<<<<<< HEAD
```

Comment:
Careful with this, shouldn't be here

Reply:
thanks for the note, still working on this.